Backend/app/provider-platform/dynamic-offer-driver-app/Main/spec/Storage/OperationHub.yaml

```diff
@@ -65,4 +67,4 @@ OperationHubRequests:
     findByDriverStatusAndType:
```

Comment:
findByCreatorStatusAndType would be more clear if we want to remove `driverId` from logic